src/app/modules/consultations/consultation-profile/response-footer/response-footer.component.ts

```diff
@@ -102,12 +102,14 @@ export class ResponseFooterComponent implements OnInit {
   vote(direction, response) {
     if (!this.loading) {
```

Comment:
@preethijayanthi Instead of adding `this.loading = true` in two blocks, you can just add it before the if statement